apps/gro/fields/address-outside.js

```diff
@@ -1,19 +1,8 @@
 'use strict';
 
 module.exports = {
-  'address-text-one': {
+  'address-text': {
     validate: ['required'],
-  },
-  'address-text-two': {
-    labelClassName: 'visuallyhidden'
-  },
-  'address-text-three': {
-    labelClassName: 'visuallyhidden'
-  },
-  'address-text-four': {
-    labelClassName: 'visuallyhidden'
-  },
-  'address-text-five': {
-    labelClassName: 'visuallyhidden'
+    'ignore-defaults': true
```

Comment:
There is a textarea in BRP that has all whitespace replaced with single spaces too - should prob be updated if you want to preserve line breaks there too